syft/workers/base.py

```diff
@@ -863,4 +873,6 @@ def create_message_execute_command(
         """
         if return_ids is None:
             return_ids = []
-        return tuple([codes.MSGTYPE.CMD, [[command_name, command_owner, args, kwargs], return_ids]])
+        return sy.Message(
```

Comment:
Again, what advantage does this give us over a tuple

Reply:
i'm working toward something which will give us more advantages - but i didn't want to make too many changes for one PR - so this is an incremental one.

Reply:
Still should import message though

Reply:
Done